packages/babel-types/scripts/generators/builders.js

```diff
@@ -28,6 +28,10 @@ function sortFieldNames(fields, type) {
   });
 }
 
+function defaultExpression(field) {
+  return Array.isArray(field.default) ? "[]" : JSON.stringify(field.default);
```

Comment:
Why do we need to special-case arrays?

Reply:
That was the behaviour of `builder.js`:
```js
arg = Array.isArray(field.default) ? [] : field.default;
```

I will check if we have weird `field.default`.

Reply:
I think we had that check to avoid sharing the same `[]` reference every time, but since it's now a new `[]` value for every call it's always a different one.